src/features/search/components/IncentiveLocationModal/IncentiveLocationModal.native.test.tsx

```diff
@@ -4,13 +4,34 @@ import { IncentiveLocationModal } from 'features/search/components/IncentiveLoca
 import { fireEvent, render, screen } from 'tests/utils'
 
 const hideModalMock = jest.fn()
+const handleActiveLocationButtonPressMock = jest.fn()
 
 describe('<IncentiveLocationModal />', () => {
   it('should call hideModal function when pressing close icon', () => {
```

Comment:
Attention, les titres des tests ne sont pas orientés utilisateurs : d'après Bruno, il vaudrait mieux qqch comme
```suggestion
  it('should hide the modal when pressing close icon', () => {
```